File: Models/PersonalInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ACA.Models
{
    public class PersonalInfo{
        public string fullName{ get; set; }
        public string gName{ get; set; }
        public string dob{ get; set; }
        public string gender{ get; set; }
        public Address address{ get; set; }
        public ContactInfo contactInfo{ get; set; }
    }
}

File: ClientApp/src/app/data-models/PersonalInfo.ts
import { Address } from "./Address";
import { ContactInfo } from "./ContactInfo";

export class PersonalInfo{
    constructor(
        public fullName : string,
        public gName: string,
        public dob: string,
        public gender: string,
        public address: Address,
        public contactInfo: ContactInfo
    ){}
}